docs/eks-installation.md

```diff
@@ -20,20 +20,14 @@ deployment yaml at:
 https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-eks.yml
 ```
 
-Based on Kubernetes service cluster IP range, adjust ``serviceCIDR`` values of antrea-agent.conf
-in antrea-eks.yml accordingly, and apply antrea-eks.yml to the EKS cluster.
-
-```bash
-kubectl apply -f antrea-eks.yaml 
 ```
-Now Antrea should be plugged into the EKS CNI and is ready to enforce NetworkPolicy.
+Note: With Antrea >=v0.9 release, you should apply `antrea-eks-node-init.yaml` before deploying Antrea.
```

Comment:
why is it a "note" instead of a required step?

Reply:
Removed *note*